Makefile

```diff
@@ -463,7 +463,7 @@ e2e-generate-xml:
 	@ hack/ci/generate-junit-xml-report.sh e2e-tests.json
 
 # Verify e2e tests compile with no errors, don't run them
-e2e-compile:
+e2e-compile: go-generate
```

Comment:
I think we may have the same issue with `e2e-local`.

```
> GO_TAGS=release make e2e-local
# github.com/elastic/cloud-on-k8s/pkg/controller/common/license
pkg/controller/common/license/check.go:41:22: undefined: publicKeyBytes
make: *** [Makefile:473 : e2e-local] Erreur 2
```

Although if in most cases we run `e2e-local` with `GO_TAGS` empty, which incidentally, I just saw is broken.

```
> GO_TAGS='' make e2e-local
package run is not in GOROOT (/home/thb/dev/tools/go/src/run)
make: *** [Makefile:473 : e2e-local] Erreur 1
```

We need to quote `$(GO_TAGS)` like you did for `'$(E2E_TAGS)'`?

Reply:
Excellent points!